contracts/WildcardSteward_v2.sol

```diff
@@ -172,6 +202,10 @@ contract WildcardSteward_v2 is Initializable {
         address payable _newReceivingBenefactor
     ) public onlyReceivingBenefactorOrAdmin(tokenId) {
         address oldBenfactor = benefactors[tokenId];
+        require(
+            oldBenfactor != _newReceivingBenefactor,
+            "Cannot change to same address"
+        );
         benefactors[tokenId] = _newReceivingBenefactor;
         benefactorFunds[_newReceivingBenefactor] = benefactorFunds[oldBenfactor];
```

Comment:
What about wiping out someone else's funds? I think it should be
```
        benefactorFunds[_newReceivingBenefactor] = benefactorFunds[_newReceivingBenefactor].add(benefactorFunds[oldBenfactor]);
```
?

Reply:
Agree with you for sure! Have made this change